src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -4510,23 +4510,33 @@ function createTextureCallsRunner<T extends Dimensionality>(
               getResult(instance_index, ${derivativeType}(0)));
 }
 
-@fragment fn fsVertex(v: VOut) -> @location(0) ${returnType} {
-  return v.result;
+@fragment fn fsVertex(v: VOut) -> @location(0) vec4u {
+  return bitcast<vec4u>(v.result);
 }
 `
       : stage === 'fragment'
       ? `
 // --------------------------- fragment stage shaders --------------------------------
+fn convertToVec4u(v: ${returnType}) -> vec4u {
```

Comment:
Is this called? is it used to convert rgba32sint to rgba32uint?

Reply:
thanks. That's not needed. Deleted